### PR TITLE
Radi4 patch 1

### PR DESCRIPTION
**Pull request by Radi4**

### CODE REVIEW

**Comment by shestakoff** [comment]
1) В чем необходимость пункта First step?
2) Зачем делать m random features? В лесу же не так
3) Не кажется ли, что у тебя как-то много баллов за это ДЗ?
4) Датасет для бустинга какой-то опасный) Год выхода по фичам, ааад)
5) calc_objective(r, y, loss = 'mse'): вот тут надо сказать, что сделать нужно для 3х фукнций потерь
6) В step(x, y, estimators, lr, loss = 'mse'): не хватает base_estimator и кажется estomators там не нужны, достаточно же r иметь
7) В GradientBoostingRegressor не хватает параметров деревьев решений
8) Для чего predict_top_n?

**Comment by shestakoff** [comment]
Задание 7-9) А какие настройки дерева? Усстанови какие-то значения по умолчанию в классе и скажи, чтобы они их придерживались?

**Comment by shestakoff** [comment]
Еще, для calc_objective и step сделай какие-нить простые тесты - должна быть ячейка с assert, чтобы они могли проверить, что правильно все сделали

**Comment by shestakoff** [comment]
И проверь что твой multiprocessing не лопнет на 1000 предикторах и что это быстрее чем по одному обучать

**Comment by shestakoff** [comment]
Кажется норм
